.github/workflows/R-CMD-check.yaml

```
@@ -56,10 +56,12 @@ jobs:
 
       - uses: r-lib/actions/setup-r-dependencies@v2
         with:
-          extra-packages: any::rcmdcheck
+          extra-packages: any::rcmdcheck, quarto-dev/quarto-r@libpaths
```

I wuld have think this needs to be 
```suggestion
          extra-packages: any::rcmdcheck, quarto=quarto-dev/quarto-r@libpaths
```

because the repo is not the same name as the package 
https://pak.r-lib.org/reference/pak_package_sources.html#package-names

But it seems it works  from the log 🤷‍♂️ 
https://github.com/r-lib/pkgdown/actions/runs/13937475969/job/39008052376?pr=2872#step:6:4775
````
  quarto         1.4.4.9004 2025-03-19 [1] Github (quarto-dev/quarto-r@1a13355)
````

Sharing just because I discovered this while doing some test like this.